docs/install/the-daemon.md

```diff
@@ -135,7 +105,7 @@ Anyone can run a cafe peer and offer services to the network. You do not need to
 [Cafe peers](/concepts/cafes) are initialized by adding some additional flags to `textile init`:
 
 ```
-textile init --seed="SSkyezjxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" --server --cafe-open
+textile init <account-seed> --server --cafe-open
```

Comment:
I didn't catch this in https://github.com/textileio/go-textile/pull/821, but this may require an update here: https://github.com/textileio/go-textile/blob/master/bin/container_daemon#L22